dune

```diff
@@ -0,0 +1,37 @@
+(dirs commands example src tests)
+
+(install
+ (section bin)
+ (package reshowcase)
+ (files
+  commands/demo-template.html
+  commands/favicon.png
+  commands/reshowcase
+  commands/ui-template.html))
+
+(rule
+ (alias start-example)
+ (deps
+  %{bin:demo-template.html}
```

Comment:
these are not binaries, you can just depend on the files

Reply:
The `reshowcase` binary (which is used in the action) depends on these files with `./`. If we just depend on the files, then `reshowcase` will crash.

Reply:
```diff
-  %{bin:demo-template.html}
+  demo-template.html
```

I believe this is what Antonio means. Should work no?

Reply:
Yes, I understand that's what he means. It won't work, because the command would be in `bin/reshowcase` and the file would be in `commands/demo-template.html`.

Reply:
Oh, I see. Keeping it as bin make sense!